### PR TITLE
Use `no-use-extend-native` plugin by default

### PR DESCRIPTION
**Pull request by kevva**
Prevents use of extended native objects.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -66,6 +66,8 @@ function handleOpts(opts) {
 		opts._config.baseConfig = 'xo/esnext';
 	}
 
+	opts._config.plugins.push('no-use-extend-native');
```
Comment (kevva) [inline index.js]: Did the easy thing and placed it here, but would you like me to set it in `DEFAULT_CONFIG` instead?

Comment (sindresorhus) [inline index.js]: No, this is fine.

**Comment by sindresorhus** [comment]
Can you set the rule config here too? I don't want to put it in https://github.com/sindresorhus/eslint-config-xo as the user would have to install it manually, and I'm planning to add more plugins, which doens't scale up there...

**Comment by sindresorhus** [comment]
Actually, nvm, it seems the config is set by default in the plugin.

**Comment by kevva** [comment]
>  ... it seems the config is set by default in the plugin.

Yup :).

**Pull request merged**